src/Sulu/Bundle/ContactBundle/Contact/AccountManager.php

```diff
@@ -238,7 +260,7 @@ public function getAccount($account, $locale)
         if ($account) {
             return $this->accountFactory->createApiEntity($account, $locale);
         } else {
-            return;
+            return null;
         }
```

Comment:
The else statement would not be needed here (as well as in other places), but I guess that's not important for this PR.